canonical-web-and-design/jaas-dashboard
# Add status list counts to headers on the model list

## Done

Add status list counts to the headers on the model list.

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- The model list should have counts along side the status.
- Also make sure the items in the model details page show statuses without the counts.

## Details
Fixes #144
